gradlew

```diff
@@ -1,7 +1,7 @@
-#!/usr/bin/env sh
```

Comment:
Should this be committed?

Reply:
Looks like different gradle versions spit out different forms of this file. But I do think this should be committed because I didn't have `+x` on my original (forgot to do it, was on Windows, derp).

Reply:
It's just a slightly newer version. In general though @bergundy gradle init scripts are always committed